RecoTracker/TkTrackingRegions/src/OuterHitPhiPrediction.h

```diff
@@ -4,32 +4,40 @@
 /** predicts phi range at a given radius r */
 
 #include "RecoTracker/TkMSParametrization/interface/PixelRecoRange.h"
-#include "RecoTracker/TkTrackingRegions/interface/TkTrackingRegionsMargin.h"
 
 #include "FWCore/Utilities/interface/Visibility.h"
 
+#include <cassert>
+
 class dso_internal OuterHitPhiPrediction {
 public:
 
-  typedef PixelRecoRange<float> Range;
-  typedef TkTrackingRegionsMargin<float> Margin;
+  using Range= PixelRecoRange<float>;
 
   OuterHitPhiPrediction( 
       const Range & phiAtVertex, 
       const Range & curvature, 
-      float originRBound, 
-      const Margin & tolerance = Margin(0,0))
+      float originRBound) 
     : thePhiAtVertex(phiAtVertex), theCurvature(curvature),
-      theOriginRBound (originRBound), theTolerance(tolerance) { } 
+      theOriginRBound (originRBound) {
+       assert(theCurvature.max()>0);
```

Comment:
@mmusich 
Thanks a lot for the explanation.
Following this, I shall change from ```auto:run2_hlt``` to ```auto:run2_hlt_relval``` for the TSG/HLTrigger side (and I assume DQM should do the same given the use cases shown in your link), so that we can then get rid of ```auto:run2_hlt``` in autoCond.py.